src/components/MaterialUI/Surfaces/SectionsExpansionPanel.test.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { mount } from "enzyme";
+import SectionExpansionPanel from "./SectionExpansionPanel";
+import ExpansionPanelMui from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import ExpansionPanelActions from "@material-ui/core/ExpansionPanelActions";
+import { ExpansionPanelProps, ExpansionPanelActionsProps } from "./expansionPanelProps";
+
+describe("Section Expansion Panel", () => {
+	it("Renders Section Expansion Panel", () => {
+		var header = <p>Header</p>;
+		var content = <p>Content</p>;
+		var actions = <p>Actions</p>;
+		var component = (
+			<SectionExpansionPanel header={header} content={content} actions={actions} />
+		);
+		var mountedComponent = mount(component);
+		var expected = (
+			<ExpansionPanelMui>
+				<ExpansionPanelSummary>{header}</ExpansionPanelSummary>
+				<ExpansionPanelDetails>{content}</ExpansionPanelDetails>
+				<ExpansionPanelActions>{actions}</ExpansionPanelActions>
+			</ExpansionPanelMui>
+		);
+
+		expect(mountedComponent.containsMatchingElement(expected), "to be truthy");
+	});
+
+	it("Fails if expansionPanelProps has wrong type", () => {
+		var component = <SectionExpansionPanel />;
+		var mountedComponent = () => {
+			component.props.expansionPanelProps = "Wrong type";
+
+			return mount(component);
+		};
+		expect(mountedComponent, "to throw");
```

Comment:
Same comment, can we test the error thrown?

Reply:
replied in previous comment

Reply:
Replaced Error with TypeError and adjust tests with this.